pkg/manager/internal.go

```diff
@@ -90,6 +91,8 @@ type controllerManager struct {
 	internalStopper chan<- struct{}
 
 	startCache func(stop <-chan struct{}) error
+
+	webhookServer *webhook.Server
```

Comment:
is manager suppose to handle lifecycle of this webhookServer, then we should probably make it a `Runnable` ?

Reply:
`*webhook.Server` IS a `Runable`.